extensions/producthunt/src/api/scraper.ts

```diff
@@ -683,7 +682,7 @@ async function scrapeDetailedProductInfo(product: Product): Promise<Product> {
                 username: postData.user.username,
                 avatarUrl: postData.user.profileImage || "",
                 profileImage: postData.user.profileImage,
-                profileUrl: `https://www.producthunt.com/@${postData.user.username}`,
+                profileUrl: `${HOST_URL}/@${postData.user.username}`,
```

Comment:
As you already have the slash in the constant, you could remove it here ;) 
```suggestion
                profileUrl: `${HOST_URL}@${postData.user.username}`,
```